src/feature/flow_chart_panel/components/play-btn/PlayBtn.tsx

```diff
@@ -23,16 +23,22 @@ const useStyles = createStyles((theme) => {
       gap: 6,
       flexWrap: "wrap",
       outline: 0,
-      padding: "padding: 8px 12px 8px 12px",
+      padding: "padding: 8px 12px px 12px",
```

Comment:
This seems incorrect. Can be modified as `padding: "8px 12px 0px 12px"` or `padding: "8px 12px 0px 12px"`